README.md

```diff
@@ -399,6 +400,24 @@ render((
         uiSchema={uiSchema} />
 ), document.getElementById("app"));
 ```
+### Array items ordering
+
+Array items are sortable by default. The `uiSchema` object spec allows you to disable ordering:
```

Comment:
*are orderable by default, and react-jsonschema-form renders move up/down buttons along them.*